packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -373,7 +373,7 @@
     border-radius: 4px;
     display: flex;
     flex-flow: column nowrap;
-    min-height: 35.8rem;
+    min-height: 29rem;
```

Comment:
you can use classNames to combine your custom classnames that has condition for not logged in height

Reply:
I directly changed here cause it seemed to resize correctly upon login...but i'll add another class just in case